src/resources/js/components/ResetPassword.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <v-container>
+    <h2 class="title">Récupération de mot de passe</h2>
+    <v-form
+        ref="form"
+        v-model="valid"
+        lazy-validation
+    >
+
+      <v-text-field
+            v-model="input.email"
+            :rules="emailRules"
+            required
+            label="Adresse email"
+      />
+
+      <v-text-field
+            v-model="input.password"
+            :append-icon="showPassword ? 'mdi-eye' : 'mdi-eye-off'"
+            :type="showPassword ? 'text' : 'password'"
+            :rules="passwordRules"
+            required
```

Comment:
your icon mdi-eye mdi-eye-off doesn't work as inteded, you forgot this line `@click:append="showPassword = !showPassword"` which makes the icon change and show the psw in clear on click

Reply:
Changed in 3a3a893